### DISCUSSION TITLE
Upcoming releases

### DISCUSSION THREAD

**Discussion by mfridman** [golang-jwt/jwt#128] [General]
Going to use this discussion for questions and updates about upcoming releases.

**Comment by mfridman** [comment]
Going to create a release from the following recent changes:

https://github.com/golang-jwt/jwt/compare/fd8cd69d..823c0140

**Comment by mfridman** [comment]
Released [`v4.2.0`](https://github.com/golang-jwt/jwt/releases/tag/v4.2.0)

https://github.com/golang-jwt/jwt/compare/fd8cd69d..c435f382

**Comment by mfridman** [comment]
Going to create a minor release and bump to `v4.3.0` in the next few days.

**Reply by mfridman** [reply]
Done. https://github.com/golang-jwt/jwt/releases/tag/v4.3.0

**Comment by mfridman** [comment]
Hey @oxisto, a few questions:

1. Are there any PR's / issues we wanted to address in the current `v4` branch?
2. Are the following 2 PR's the best place to start reviewing upcoming `v5` changes?

https://github.com/golang-jwt/jwt/pull/234
https://github.com/golang-jwt/jwt/pull/236

I have some cycles coming up in the next ~month or so.

**Reply by oxisto** [reply]
> Hey @oxisto, a few questions:
> 
> 1. Are there any PR's / issues we wanted to address in the current `v4` branch?

No, I think all is merged

> 2. Are the following 2 PR's the best place to start reviewing upcoming `v5` changes?
> 
> #234 #236

Yes

> 
> I have some cycles coming up in the next ~month or so.

**Comment by oxisto** [comment]
@mfridman the `v5` pre-release PR is now on `main`. So now would probably be a good time to do a alpha/beta/rc1 pre-release of `v5` based on the latest commit in main.

**Reply by mfridman** [reply]
Sounds good to me, do you want me to do it, or will you? I'm good either way, just checking so we don't duplicate the release.

**Reply by oxisto** [reply]
You are Mr. Release ;)

**Reply by mfridman** [reply]
Alright, this is done. I really liked your migration guide, so I just dumped that into the release notes.

Folks can continue to use either `v4` or `v5` or the non-supported `v3.2.2+incompatible`. So we should be okay moving forward.

```sh
go: downloading github.com/golang-jwt/jwt/v4 v4.5.0
go: added github.com/golang-jwt/jwt/v4 v4.5.0
```

```
go: downloading github.com/golang-jwt/jwt/v5 v5.0.0-rc.1
go: added github.com/golang-jwt/jwt/v5 v5.0.0-rc.1
```

**Comment by oxisto** [comment]
Released `v5.0.0`: https://github.com/golang-jwt/jwt/releases/tag/v5.0.0

**Comment by mfridman** [comment]
@oxisto we should probably do a release soon.
